tests/libwallet_api_tests/main.cpp

```diff
@@ -1146,10 +1149,10 @@ int main(int argc, char** argv)
     TESTNET_WALLET5_NAME = WALLETS_ROOT_DIR + "/wallet_05.bin";
     TESTNET_WALLET6_NAME = WALLETS_ROOT_DIR + "/wallet_06.bin";
 
-    CURRENT_SRC_WALLET = TESTNET_WALLET6_NAME;
-    CURRENT_DST_WALLET = TESTNET_WALLET5_NAME;
+    CURRENT_SRC_WALLET = TESTNET_WALLET5_NAME;
+    CURRENT_DST_WALLET = TESTNET_WALLET1_NAME;
```

Comment:
Why are these changing all the time ? I assume because the use live wallets that just happen to have enough or not enough monero, so if this is true, this should change at some point so tests become usable by everyone. A job for later though.